django/db/models/fields/related.py

```diff
@@ -32,6 +20,22 @@
 from django.utils.translation import ugettext_lazy as _
 from django.utils.version import get_docs_version
 
+from . import (
+    AutoField, Field, IntegerField, PositiveIntegerField,
+    PositiveSmallIntegerField,
+)
+from .related_descriptors import (
+    ForwardManyToManyDescriptor, ForwardManyToOneDescriptor,
```

Comment:
The M2M descriptor is used both for the forward and reverse relation. We should probably drop the `Forward` part.